source/crud/delete.txt

```diff
@@ -51,6 +51,14 @@ To delete all documents from a collection, pass an empty
    more information and examples, see
    :method:`~db.collection.deleteMany()`.
 
+.. note::
+
+   If you want to delete all documents from a large collection, dropping
+   and recreating the collection may have faster performance than using
+   the :method:`db.collection.deleteMany()` method with an empty filter.
+   When you recreate the collection, you must also recreate any
+   specified collection characteristics. 
```

Comment:
It might be good to explicitly mention that collection indexes will also need to be recreated.

Reply:
Done!